client/gutenberg/editor/controller.js

```diff
@@ -211,16 +151,13 @@ export const redirect = async ( context, next ) => {
 	}
 
 	const postType = determinePostType( context );
-	if ( ! shouldLoadGutenframe( state, siteId, postType ) ) {
-		const postId = getPostID( context );
+	const postId = getPostID( context );
 
-		const url = postType
-			? getEditorUrl( state, siteId, postId, postType )
-			: getSiteEditorUrl( state, siteId );
-		// pass along parameters, for example press-this
-		return window.location.replace( addQueryArgs( context.query, url ) );
-	}
-	return next();
```

Comment:
Since we no longer run the next middleware, I guess we can remove the `authenticate` and `redirectPostEditor` middlewares too?

https://github.com/Automattic/wp-calypso/blob/c076610bc6fbb5fdc78fe95b405b37cb3771852c/client/gutenberg/editor/index.js#L22-L23